subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -215,7 +245,7 @@ Naturally when the task is executed again with no changes, then the entire task
 include::{samplesPath}/userguide/tasks/incrementalTask/incrementalTaskNoChange.out[]
```

Comment:
I know this isn't part of the change, but this file is empty. Can we not have it print _something_ out? It just looks like a broken example right now.

Reply:
@wolfs :point_up: (in case you don't get a notification from the above comment)

Reply:
I guess if we run the build with `--verbose` we will get the task headers, right?